examples/sklearn_elasticnet_wine/conda.yaml

```
@@ -1,11 +1,9 @@
 name: tutorial
 channels:
   - defaults
-  - anaconda
-  - conda-forge
 dependencies:
   - python=3.6
-  - scikit-learn=0.19.1
+  - scikit-learn=0.23.2
   - pip
   - pip:
     - mlflow>=1.0
```

```suggestion
  - pip
  - pip:
    - scikit-learn=0.23.2
    - mlflow>=1.0
```

Can we change `scikit-learn`  to a pip dependency?